inference-engine/tests/functional/inference_engine/ir_serialization/serialize.cpp

```diff
@@ -67,10 +67,11 @@ INSTANTIATE_TEST_CASE_P(IRSerialization, SerializationTest,
                         std::make_tuple("experimental_detectron_detection_output.xml", ""),
                         std::make_tuple("experimental_detectron_detection_output_opset6.xml", ""),
                         std::make_tuple("nms5.xml", "nms5.bin"),
-                        std::make_tuple("shape_of.xml", "")));
+                        std::make_tuple("shape_of.xml", ""),
+                        std::make_tuple("conv_with_rt_info.xml", "")));
```

Comment:
You have also .bin for this model so we can make it explicit here: std::make_tuple("conv_with_rt_info.xml", "conv_with_rt_info.bin")));